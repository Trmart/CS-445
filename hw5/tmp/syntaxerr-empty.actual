====================================
ERROR(SYSTEM): niceTokenStr fails to find string 'en'
