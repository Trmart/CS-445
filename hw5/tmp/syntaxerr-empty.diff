2,5c2
< ERROR(2): Syntax error, unexpected end of input, expecting "bool" or "char" or "int" or identifier.
< FILE: syntaxerr-empty.c-
< Number of errors: 1
< Number of warnings: 0
---
> ERROR(SYSTEM): niceTokenStr fails to find string 'en'
