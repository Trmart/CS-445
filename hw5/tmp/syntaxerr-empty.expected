====================================
ERROR(2): Syntax error, unexpected end of input, expecting "bool" or "char" or "int" or identifier.
FILE: syntaxerr-empty.c-
Number of errors: 1
Number of warnings: 0
